src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
```

Comment:
I recommend removing indents between lines of code, for better readability. To follow the logic of indentation - separation of elements in the file.